node/src/actors/peers_manager/mod.rs

```diff
@@ -141,6 +144,15 @@ impl PeersManager {
         });
     }
 
+    /// Retrieve current check_melted_peers period
+    pub fn current_check_melted_peers_period(&self) -> Duration {
+        if self.peers.bootstrapped {
+            self.check_melted_peers_period
+        } else {
+            Duration::from_secs(60)
+        }
+    }
+
     /// Method to periodically melt peers
     fn melt_peers(&self, ctx: &mut Context<Self>, check_melted_peers_period: Duration) {
```

Comment:
If the period is calculated using `act.current_check_melted_peers_period()`, you don't need this argument.

```suggestion
    fn melt_peers(&self, ctx: &mut Context<Self>) {
```